src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/RedirectHandler.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
```

Comment:
This include was added to use [CallerMemberName]. I'll take that feature out if anyone thinks this will be problematic.

Reply:
It's fine :)